samples/caf_sensor_manager/sample.yaml

```diff
@@ -18,6 +18,23 @@ tests:
         - "main state:READY"
         - "Send sensor buffer desc address:"
         - "sensor_data_aggregator_release_buffer_event"
+  sample.caf_sensor_manager.sensor_stub.correctness_test:
+    build_only: false
+    harness: console
+    platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp qemu_cortex_m3
```

Comment:
How is this gonna work with quemu platform if it has one core?

Reply:
Why one platform is a problem here? It works on 52840 which also has one core.